targets/WindowsSdk/source/source/PlayFabSettings.cpp.ejs

```diff
@@ -2,14 +2,14 @@
 
 namespace PlayFab
 {
-    const utility::string_t PlayFabSettings::sdkVersion = U("<%- sdkVersion %>");
-    const utility::string_t PlayFabSettings::buildIdentifier = U("<%- buildIdentifier %>");
-    const utility::string_t PlayFabSettings::versionString = U("WindowsSDK-<%- sdkVersion %>");
+    const utility::string_t PlayFabSettings::sdkVersion = L"<%- sdkVersion %>";
```

Comment:
I think utility::string_t comes from CppRestSDK. It is the same as std::wstring, right? Any reason we are not just using std::wstring?

Reply:
CppRestSdk, correct. I just use the string it wants because I don't want to constantly re-cast it to the string type it wants. I also worry about memory use/efficiency of constantly copying strings. If I use the string it wants, there's no conversion.
And for the most part, only CppRestSdk sees... MOST of the utility strings.

In the end I only really care about not abusing the memory with lots of conversions. If that's better than my worst-case-assumptions about the situation, then I don't care.